packages/Input/Input.modules.scss

```diff
@@ -11,3 +11,9 @@
   composes: absolute centerVertically from '../../shared/styles/Position.modules.scss';
   composes: iconRight from '../../shared/components/FormField/FormField.modules.scss';
 }
+
+.input {
+  min-height: 3.25rem;
+  max-height: 3.25rem;
+  height: auto;
```

Comment:
Is there a specific reason why the `height` prop is set? Because setting `min-height` and `max-height` should pretty much be sufficient.

Reply:
The issue is specifically with `height`. In order to get around it we need to clear the existing height with `height: auto` and instead use `min-height` and `max-height`.